plugins/ltac/g_rewrite.mlg

```diff
@@ -104,6 +97,11 @@ END
 GRAMMAR EXTEND Gram
   GLOBAL: rewstrategy;
   rewstrategy:
+    [ NONA
+      [ IDENT "fix"; id = identref; ":="; s = rewstrategy1 -> { StratFix (id, s) }
+      | h = ne_rewstrategy1_list -> { h } ] ]
+  ;
+  ne_rewstrategy1_list:
```

Comment:
There is a little ambiguity in using `ne_rewstrategy1_list`: it looks generic but actually has `;` as a separator. Maybe use `ne_rewstrategy1_list_sep_semicolon` instead (or back to `rewstrategy1_seq` or `rewstrategy1_compose`), if ever it can be changed quickly with a global replacement.

Reply:
Updated.  It can be changed with `git grep --name-only ne_rewstrategy1_list_sep_semicolon | xargs sed -i 's/ne_rewstrategy1_list_sep_semicolon/<anything_else>/g'`